en/starter/generator.md

```diff
@@ -81,6 +81,13 @@ On Windows, use this command:
 > set DEBUG=myapp:* & npm start
 ```
 
+Or in Windows PowerShell, use this command:
+
+```sh
+> set DEBUG=myapp:* | npm start
```

Comment:
So this command doesn't produce the expected debug output when run. It should show the line of what port it's listening on. This makes me believe that, even though now it doesn't _error_, it's still not the correct command to be running in PowerShell.